### DISCUSSION TITLE
Docker monitoring finds undefined version

### DISCUSSION THREAD

**Discussion by renan-alm** [renovatebot/renovate#12830] [Request Help]
Hi,

I'm currently using a self-hosted instance of Renovate to monitor Docker images into Dockerfiles and at the moment I cannot make Renovate be able to find my docker versions.

If I run towards an image that has semver `1.33` then I get a newer version. However, when I have a current version that follows for instance `v10`, then Renovate can't find the newer version.
`FROM hub-example.com/AppExample:1.33`
`FROM hub-example.com/AppExample:v10`

At the moment my package rule looks like this:      
```
        {
            "groupName": "docker-deps",
            "matchDatasources": ["docker"],
            "matchManagers": ["dockerfile"],
            "registryUrls": [
                "https://hub-example.com"
            ],
            "versioning": "maven"
        }
```

And the logs are looking like this:
```
DEBUG: getLabels(https://hub-example.com, images/appexample, undefined) (repository=project/appexample)
DEBUG: getManifestResponse(https://hub-example.com, images/appexample, undefined) (repository=project/appexample)
 .
 .
 .
DEBUG: Docker Manifest is unknown (repository=project/appexample)
```

And again, this works all fine when I use an image that uses semver. 
Can you please give me some directions on what could be wrong?

Kind regards,
Renan A.

**Comment by viceice** [comment]
renovate can't update this by default, you need to manually switch to a compatible version

**Reply by renan-alm** [reply]
I see, but is it possible to be done using a regex? Maybe something like `datasourceTemplate` or `versioningTemplate`?

I've tried something like this `"regex:^v(?<minor>\\d+)$"` but no success.

**Reply by viceice** [reply]
The Regex needs to match all versions
